coding/trie_builder.hpp

```diff
@@ -16,13 +17,13 @@
 // pre-order alphabetically reversed (parent, last child, first child).
 
 // Leaf node format:
-// [value] ... [value]
+// [valueList]
 
 // Internal node format:
 // [1: header]: [2: min(valueCount, 3)] [6: min(childCount, 63)]
```

Comment:
Is it still correct? Are you going to store valueCount separately (in header) from valueList?